internal/cmd/plugin/command_install.go

```diff
@@ -0,0 +1,304 @@
+package plugin
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"strings"
+
+	"github.com/go-yaml/yaml"
+	"github.com/hashicorp/go-version"
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/exec"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/types"
+	"github.com/confluentinc/cli/internal/pkg/utils"
+)
+
+const (
+	Go     = "go"
+	Python = "python"
+	Shell  = "shell"
+)
+
+const (
+	programNotFoundMsg      = "[WARN] Unable to find %s. Check that it is installed in a directory in your $PATH.\n"
+	unableToParseVersionMsg = "[WARN] Unable to parse %s version.\n"
+	insufficientVersionMsg  = "[WARN] Installed %s version %s is less than the required version %s.\n"
+)
+
+func (c *command) newInstallCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "install",
+		Short: "Install official Confluent CLI plugins.",
+		Long:  "Install official Confluent CLI plugins from the confluentinc/cli-plugins repository.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.install,
+	}
+
+	cmd.Flags().String("plugin-directory", "", `The plugin installation directory. This must be a directory in your $PATH. If not specified, the default is the CLI installation directory for Windows, and "usr/local/bin" for MacOS and Linux.`)
```

Comment:
What if we installed all plugins to `~/.confluent/plugins` and make sure to scan that directory even if it isn't part of $PATH?

Reply:
Oh, I like that a lot.

That will let me remove a chunk of code, too.